index.bs

```diff
@@ -1152,16 +1163,21 @@ dictionary FilePickerAcceptType {
 dictionary FilePickerOptions {
     sequence<FilePickerAcceptType> types;
     boolean excludeAcceptAllOption = false;
+    USVString id;
+    StartInDirectory? startIn;
 };
 
 dictionary OpenFilePickerOptions : FilePickerOptions {
     boolean multiple = false;
 };
 
 dictionary SaveFilePickerOptions : FilePickerOptions {
+    USVString? suggestedName;
 };
 
 dictionary DirectoryPickerOptions {
+    USVString id;
```

Comment:
Aside: Yay for USVString; do we have tests with a lone surrogate being mapped to U+FFFD ?